.github/reviewers.yaml

```diff
@@ -0,0 +1,37 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - maintainers
+
+  groups:
+    maintainers:
+      - nrb
+      - ashish-amarnath
+      - carlisia
+      - zubron
+      - dsu-igeek
+      - jenting
+
+    tech-writer:
+      - a-mccarthy 
+
+files:
+  'public/**':
+    - tech-writer
```

Comment:
Is this referring to files under `$(REPO_ROOT)/public`?
We don't have a `$(REPO_ROOT)/public` directory in our repo.
Is this leftover from copy-pasta?

Reply:
Yep. Oversight!